core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -106,8 +106,11 @@
 
   // basic configuration
   private final ClientContext context;
+  private static double MAX_MEM_PERCENTAGE = 0.98;
   private final long maxMem;
+  private static long MAX_MEMORY_DIVISOR = 20;
```

Comment:
What does this divisor represent, other than a portion of a division problem? Is there a better name we can use for these two constants?

Also, probably cleaner style to put the static class constants together above the instance member variables.